python/py-beancount/Portfile

```diff
@@ -4,18 +4,22 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-beancount
+categories-append   finance
 version             2.2.1
+revision            1
 checksums           rmd160  60650715322a8f302455f3028400281f23d850d2 \
                     sha256  ebcb59bd7c0e18a858c55d6c30eacee3fa949ee5d2c452a7aa80690e36ae2f77 \
                     size    597196
 
 license             GPL-2
 platforms           darwin
-supported_archs     noarch
-maintainers         nomaintainer
+maintainers         {blair @blair} openmaintainer
```

Comment:
Please don't add me as a maintainer as I don't use this package any more.

Reply:
Done.